litmus-portal/frontend/src/views/Analytics/ApplicationDashboards/Steps/TuneTheQueries/EditPanelsWizard/index.tsx

```diff
@@ -240,7 +242,9 @@ const EditPanelsWizard: React.FC<EditPanelsWizardProps> = ({
         grids: true,
         left_axis: true,
       },
-      panel_name: 'Untitled Metric',
+      panel_name: t(
+        'analyticsDashboard.applicationDashboards.tuneTheQueries.untitledPanel'
```

Comment:
Please don’t add translations here as we support only English for all the pre defined group and panel names

Reply:
We should add transation for all dashboards and for all  supported languages. cc: @arkajotiMukherjee

Reply:
Then please add the same for all panel names and group names for the pre defined dashboards @whitetiger1399

Reply:
Temporary fix